requirements.txt

```diff
@@ -1,4 +1,4 @@
-networkx>=2.0,<2.3
+# networkx==2.2
```

Comment:
Please don't do strict version locking, allow the room to accomodate small changes like done before: ">=2.0,<2.3"

Also, It looks like you've commented networkx out!

Reply:
yes I have commented it out, i am adding it in the requirements from the setup.py file as the version needs to be checked